custom_components/volkswagencarnet/__init__.py

```diff
@@ -51,9 +46,14 @@
 
 
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
-    """Setup Volkswagen Carnet component"""
+    """Setup Volkswagen WeConnect component"""
+
+    if entry.options.get(CONF_SCAN_INTERVAL):
+        update_interval = timedelta(minutes=entry.options[CONF_SCAN_INTERVAL])
+    else:
+        update_interval = timedelta(minutes=DEFAULT_UPDATE_INTERVAL)
```

Comment:
```suggestion
        update_interval = DEFAULT_UPDATE_INTERVAL
```

Reply:
Found this single bug: `DEFAULT_UPDATE_INTERVAL` is already of type `timedelta`.

Reply:
Thanks. Did a slight different fix, but it should resolve this.